src/System.Xml.XDocument/System/Xml/Linq/XComment.cs

```diff
@@ -92,6 +94,21 @@ public override void WriteTo(XmlWriter writer)
             writer.WriteComment(value);
         }
 
+        /// <summary>
+        /// Write this <see cref="XComment"/> to the passed in <see cref="XmlWriter"/>.
+        /// </summary>
+        /// <param name="writer">
+        /// The <see cref="XmlWriter"/> to write this <see cref="XComment"/> to.
+        /// </param>
+        /// <param name="cancellationToken">A cancellation token.</param>
+        public override async Task WriteToAsync(XmlWriter writer, CancellationToken cancellationToken)
+        {
+            if (writer == null) throw new ArgumentNullException("writer");
```

Comment:
The single-line test/throw style in my patch is consistent with existing XLINQ code. Do you still prefer me to use the new guidelines within my patch, or should we hold off style changes for another PR?